### ISSUE TITLE
Pitch names's color don't change with inverted colors

### ISSUE THREAD

**Issue by Thiago-Morais** [musescore/MuseScore#20312]
### Issue type

UI bug (incorrect info or interface appearance)

### Bug description

When you are using "pitch names" and "Invert Score", the notes turn white as they should but the pitch names remain white as well instead of turning black as they should.
That makes it impossible to read the pitch name.
When "Invert Score" is enabled, the black notes should turn white and the white pitch names should turn black.

### Steps to reproduce

1) Add notes
2) Enable "pitch names" on "Staff/Part Properties"
3) Enable "Invert Score" on the "Apperance" settings

### Screenshots/Screen recordings

![Captura de tela 2022-12-29 055642](https://github.com/musescore/MuseScore/assets/21369746/d99d9cc7-99ea-4833-af44-426593340c5c)
![Captura de tela 2022-12-29 055649_0](https://github.com/musescore/MuseScore/assets/21369746/edb9fb69-18da-4ffe-85ce-dc31a1890874)

### MuseScore Version

4.1.1.232071203

### Regression

I don't know

### Operating system

Windows 10 Pro

**Comment by cbjeukendrup** [comment]
Technically closely related to #11570

**Comment by Jojo-Schmitz** [comment]
Isn't this a duplicate of #16506? Or at least also closely related?

**Comment by cbjeukendrup** [comment]
Not really. This one is specifically about the note names on the notes, and not the notes themselves.
